swarm/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.44.1 - unreleased
```

Comment:
```suggestion
## 0.44.1
```

Slight (and IMO exciting) change in process: https://github.com/libp2p/rust-libp2p/pull/4894